armi/reactor/blueprints/tests/test_materialModifications.py

```diff
@@ -189,14 +189,56 @@ def test_filterMaterialInput(self):
             "fuel2": {"ZR_wt_frac": 0.3, "U235_wt_frac": 0.3},
         }
         componentDesign = a[0][0]
-        filteredMaterialInput = BlockBlueprint._filterMaterialInput(
+        filteredMaterialInput, _ = BlockBlueprint._filterMaterialInput(
```

Comment:
Can we might as well test whatever `byComponentMatModKeys` get returned here?

It seems like a low-hanging fruit additional test, since your function now returns more info.

Sorry to bother!